src/session/session_api.c

```diff
@@ -2735,3 +2757,11 @@ __wt_open_internal_session(WT_CONNECTION_IMPL *conn, const char *name, bool open
     *sessionp = session;
     return (0);
 }
+
+#ifdef HAVE_UNITTEST
```

Comment:
This probably falls under the Q3 testing scope, but is there a way we can expose these functions to unit testing without writing one wrapper per function? 
I suspect the end result of this solution will be one wrapper function for every static function in the code base.

Reply:
cc: @korteland, @jeremythorp. Andrew is correct that at some point we will have hundreds of these wrapper. I don't think there is a way around them. Are any other methods possible?

Reply:
An idea: for every file file.c that contains static functions have a file file_wrappers.h that contains all the static function wrappers. File file.c would include file_wrappers.h at the end. This way the wrappers do not get in the way while reading file file.c.

Reply:
We could also look at a `WT_STATIC` macro which behaves like `WT_INLINE` and is removed by the pre-processer when we compile with a certain flag. I think we've discussed this option in the past but I don't remember why we didn't pursue it.

Reply:
@korteland and I introduced the __ut_ wrapper functions as a minimal, if not perfect, change to expose these functions for unit testing. I'm more than happy to consider other options that are cleaner but required more changes (such as splitting the current headers into multiple more focussed ones). The key thing is that internal functions need to be exposed for unit/sub-API testing.